models/repo.go

```diff
@@ -1903,6 +1905,128 @@ func (repos MirrorRepositoryList) LoadAttributes() error {
 	return repos.loadAttributes(x)
 }
 
+type ScrubSensativeDataOptions struct {
+	LastCommitID	string
+	CommitMessage	string
+}
+
+// SrubSenstaiveData removes emails and passwords from the manifest.json or project.json file's history.
```

Comment:
Yes, we are learning I can't spell Sensitive to save my life.....and I'm very sensative about it. 😛